docs/contributing/index.rst

```diff
@@ -3,59 +3,33 @@
 Contributing to Weblate
 =======================
 
+Language
+--------
 
-There are dozens of ways to contribute in Weblate. Any help is welcomed, be it
-coding, graphics design, documentation or sponsorship:
+`Translate <https://hosted.weblate.org/projects/weblate/>`_ Weblate itself,
+and point out errors in source strings while you do. Suggest source string changes, or
+click "Source string location" in the lower right of the regular editor to have
+a go yourself.
 
-* :doc:`issues`
-* :doc:`start`
-* :ref:`translating-weblate`
-* :ref:`contributing-docs`
-* :ref:`funding-weblate`
+Here in the documentation, click :guilabel:`Edit on GitHub` atop the page to do the same thing.
+`Open an issue https://github.com/WeblateOrg/weblate/issues/new?assignees=comradekingu&labels=ux&template=bug_report.md>`_
+if you know something is wrong or have an issue with the documentation, but not how to fix it.
+Remove the template stuff; a link and description will do.
 
-.. _translating-weblate:
+Graphics
+--------
+The visuals are in https://github.com/WeblateOrg/graphics
 
-Translating Weblate
--------------------
+`Simple Icons issue https://github.com/simple-icons/simple-icons/issues/5085>`_
+`Fork Awesome issue https://github.com/ForkAwesome/Fork-Awesome/issues/218#issuecomment-611892888>`_
```

Comment:
I don't see a reason for including these links - there is nothing to be done there (one is finished, just cannot be released for whatever reasons on their side and one is on hold until we get more starts).

Reply:
It is a fantastic example of how to improve Weblate in other venues. Just adding a star is maybe the best example of the simplest thing to do, with somewhat of an impact.

Reply:
Sorry, but I don't think it's reasonable to expect people to star a repo because they read an issue which is linked in the documentation. Starting might be the simplest thing to do, but figuring it out this way is definitely not.